src/Gui/Application.cpp

```diff
@@ -2180,6 +2180,22 @@ void Application::runApplication()
 
             int major = context.format().majorVersion();
             int minor = context.format().minorVersion();
+            if (major < 2) {
+                auto version = QLatin1String(App::Application::Config()["ExeVersion"].c_str());
+                auto msgBoxTitle = Gui::GUISingleApplication::applicationName()
+                    + QString::fromUtf8(" ")
+                    + version
+                    + QString::fromUtf8(" ")
+                    + QObject::tr("Invalid OpenGL Version");
+                auto message =
+                    QObject::tr("This system is running OpenGL %1.%2.").arg(major).arg(minor)
+                    + QString::fromUtf8(" ")
+                    + QObject::tr("FreeCAD requires OpenGL 2.0 or above.")
+                    + QString::fromUtf8(" ")
+                    + QObject::tr("Please upgrade your graphics driver and/or card as required.");
```

Comment:
This is not a big deal at all, but is there any real reason why these strings need to be separated in chunks? When translating I feel like it would be better if they are all together.

Reply:
I agree, but the line length gets annoyingly long. Do you have a good technique for mitigating that?

Reply:
The problem (from my perspective looking at translations on crowdin) with having large string chunks is that they include newlines. Translators sometimes get 'creative' with newlines or just ignore them altogether. So the strategy is to limit newlines in translations as best as possible. But obviously inevitable.

Reply:
point taken, out of curiosity what do we do if we do need long text translated, would syntax like this work:
```c++
                QObject::tr("This system is running OpenGL %1.%2. "
                    "FreeCAD requires OpenGL 2.0 or above. "
                    "Please upgrade your graphics driver and/or card as required."
                    ).arg(major).arg(minor);
```

Reply:
Yes - it works fine actually. In Python it doesn't, but that's because the Python lupdate script doesn't actually run a Python interpreter, so doesn't correctly concatenate the strings. In C++ it works fine, so I have updated this PR.

Reply:
> The problem (from my perspective looking at translations on crowdin) with having large string chunks is that they include newlines. Translators sometimes get 'creative' with newlines or just ignore them altogether. So the strategy is to limit newlines in translations as best as possible.

I've put the newline in this paragraph as a separate (non-translated) string, following our practice elsewhere. So I think the translators will be happy with what they see now.